### PR TITLE
[WIP] poc/research bonding curve agreement

### PR DESCRIPTION
**Pull request by diminator**
## Description

Add a description of your changes here.

## Is this PR related with an open issue?

Related to Issue #

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
 
## Checklist:

- [ ] Follows the code style of this project.
- [ ] Tests Cover Changes
- [ ] Documentation

#### Funny gif

Put a link of a funny gif inside the parenthesis-->

### CODE REVIEW

File: contracts/bondingcurves/OceanBondingCurve.sol [hunk 1]
```diff
@@ -0,0 +1,346 @@
...
+        oceanToken = OceanToken(_tokenAddress);
+        // initialize variables
+        scale = 1;
+        reserveRatio = 700000;
+        // MAX_WEIGHT = 1000000;
+        gasPrice = 0 wei;
+    }
+
+    // Testing: request initial fund transfer
+    function requestTokens(uint256 amount) public returns (uint256) {
```
Comment (gongf05) [inline contracts/bondingcurves/OceanBondingCurve.sol]: This function is for development purpose. It should be removed in production code.

Comment (aaitor) [reply contracts/bondingcurves/OceanBondingCurve.sol]: All the code included in keeper contracts project must be production code, if it's not must be removed from the repo

Comment (diminator) [reply contracts/bondingcurves/OceanBondingCurve.sol]: I was thinking about to create a separate branch for these types of trials
For example: `research` or `experimental` or `future`?

It's not sure if these things will go into production, but having them in this repo helps end-to-end development of new features

Comment (aaitor) [reply contracts/bondingcurves/OceanBondingCurve.sol]: new branches is okay. I would create a `fork` with name `keeper-experimental` where we can do whatever we want

Comment (diminator) [reply contracts/bondingcurves/OceanBondingCurve.sol]: that sounds good

**Comment by diminator** [comment]
closed in favor of https://github.com/oceanprotocol/keeper-experimental/pull/1

**Pull request closed without merging**